pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -72,6 +77,16 @@ func NewFramework(r Registry, _ *runtime.Unknown) (Framework, error) {
 	return f, nil
 }
 
+// QueueSortFunc returns the function to sort pods in scheduling queue
+func (f *framework) QueueSortFunc() LessFunc {
+	if len(f.queueSortPlugins) == 0 {
+		return nil
+	}
+
+	// only active one at a time.
+	return f.queueSortPlugins[0].Less
```

Comment:
An alternative to this approach is to support multiple plugins for QueueSort. We could rename `Less` function to `Cmp` and let it return -1, 0, or 1. A returned 0 means that the values are equal. In that case, we can call the next plugin Cmp until one returns non-zero. For now, I think we can stay with the current implementation.